pkgs/by-name/bo/bootc/package.nix

```diff
@@ -0,0 +1,42 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  libz,
+  zstd,
+  pkg-config,
+  openssl,
+  glib,
+  ostree,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "bootc";
+  version = "1.1.0";
+  cargoHash = "sha256-2Ka3n6sT1RUlReNjtV6tMe4ow/M7DFNvWPZktwcEi/w=";
+
+  src = fetchFromGitHub {
+    owner = "containers";
+    repo = "bootc";
+    rev = "v${version}";
+    hash = "sha256-vsJwJHPE5Puv9xCnDRDtHzNuFj1U7s1HzZ2vQKTavhs=";
+  };
+
+  nativeBuildInputs = [ pkg-config ];
+
+  buildInputs = [
+    libz
+    zstd
+    openssl
+    glib
+    ostree
+  ];
+
```

Comment:
Would be good to add https://nixos.org/manual/nixpkgs/unstable/#versioncheckhook if possible.